ci/ci/utils.py

```diff
@@ -36,3 +37,62 @@ async def add_deployed_services(
 ''',
         [(namespace, service) for service in services],
     )
+
+
+def generate_gcp_service_logging_url(
+    project: str,
+    services: List[str],
+    namespace: str,
+    start_time: str,
+    end_time: Optional[str],
+    severity: Optional[List[str]],
+):
+    service_queries = []
+    for service in services:
+        service_queries.append(
+            f'''
+(
+resource.type="k8s_container"
+resource.labels.namespace_name="{namespace}"
+resource.labels.container_name="{service}"
+)
+'''
+        )
+
+    query = ' OR '.join(service_queries)
+
+    if severity is not None:
+        severity_queries = []
+        for level in severity:
+            severity_queries.append(f'severity={level}')
+        query += ' OR '.join(severity_queries)
+
+    timestamp_query = f';startTime={start_time}'
+    if end_time is not None:
+        timestamp_query += f';endTime={end_time}'
+
```

Comment:
Nit: the severity, timestamp and URL construction I believe are the same between these two functions. Can you separate those out into a helper?